package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "guardian-management-client",
-  "version": "2.0.0",
+  "name": "node-guardian",
+  "version": "2.0.1",
```

Comment:
You have to also push the tag so I usually bump the version separately. Actually at some point we should do this on the CI,